content/sensu-core/2.0/reference/entities.md

```diff
@@ -14,21 +14,21 @@ menu:
 
 ### Why "Entities"?
 
-Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless". Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
+Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless", so we've adjusted the nomenclature accordingly. Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
 
 ### What is a Sensu entity?
 
-Entities, formally known as clients in Sensu 1.x, are monitoring agents (e.g. server, container, etc) that need to be monitored. They contain system information such as the hostname, OS, platform, and version. Entities not only provide context to event data but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
+An Entity, formally known as a client in Sensu 1.x, represents anything (e.g. server, container, network switch, etc) that needs to be monitored. It can contain system information such as the hostname, OS, platform, and version. An Entity not only provides context to event data (what/where the event is from) but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
 
 ## Agent entities
 
 ### What is an agent entity?
 
-An agent entity is a Sensu entity which falls under the "agent" class. Agent entities are monitoring agents, which are installed and run on every system that needs to be monitored. The entity is responsible for registering the system with Sensu, sending keepalive messages (the Sensu heartbeat mechanism), and executing monitoring checks. Each entity is a member of one or more subscriptions – a list of roles and/or responsibilities assigned to the system (e.g. a webserver, database, etc). Sensu entities will "subscribe" to (or watch for) check requests published by the Sensu server (via the Sensu Transport), execute the corresponding requests locally, and publish the results of the check back to the transport (to be processed by a Sensu server).
+Agent entities are monitoring agents, which are installed and run on every system that needs to be monitored. The entity is responsible for registering the system with Sensu, sending keepalive messages (the Sensu heartbeat mechanism), and executing monitoring checks. Each entity is a member of one or more subscriptions – a list of roles and/or responsibilities assigned to the system (e.g. a webserver, database, etc). Sensu entities will "subscribe" to (or watch for) check requests published by the Sensu server (via the Sensu Transport), execute the corresponding requests locally, and publish the results of the check back to the transport (to be processed by a Sensu server).
```

Comment:
Since we're being super clear here, maybe change "registering the system with Sensu" to acknowledge the specific service it registers with, such as "registering the system with Sensu backend service" if that's accurate.  (I need your help there 😄). 

Love the additional explanation of the heartbeat mechanism - great details in here 👍

Reply:
Maybe also: 

- bold **subscriptions** since that's a new concept we want to highlight

Reply:
> subscriptions – a list of roles and/or responsibilities assigned to the system

Are these assigned to the system or, more specifically, to the agent entity? I'm not sure what "the system" means in this context.